CMakeLists.txt

```diff
@@ -193,14 +193,8 @@ KOKKOS_SETUP_BUILD_ENVIRONMENT()
 
 OPTION(BUILD_SHARED_LIBS "Build shared libraries" OFF)
 
-SET(KOKKOS_EXT_LIBRARIES Kokkos::kokkos Kokkos::kokkoscore Kokkos::kokkoscontainers Kokkos::kokkosalgorithms)
```

Comment:
why is this gone?

Reply:
what was this used for at all?

Reply:
Never used (I checked in Trilinos too)

Reply:
Also gripped for `_EXT_LIBRARIES` and packages don't define it

Reply:
It is used [here](https://github.com/dalg24/kokkos/blob/develop/cmake/KokkosConfig.cmake.in#L12)

Reply:
Sorry it was from an old commit. We don't use this anymore